### DISCUSSION TITLE
"type 'List<Object?>' is not a subtype of type 'PigeonUserDetails?' in type cast".

### DISCUSSION THREAD

**Discussion by PRAM1N** [firebase/flutterfire#16859] [Q&A]
is:issue is:open "type 'List<Object?>' is not a subtype of type 'PigeonUserDetails?' in type cast". Updating firebase_auth & firebase_core can solve this error, so I want firebase_auth & firebase_core to latest version, keeping cloud_firestore, geoflutterfire2, firebase_messaging, firebase_storage, firebase_core_platform_interface compatible with them. Someone help me with it

This is my pubspec.yaml file
name: 
description: 

# Prevent accidental publishing to pub.dev.
publish_to: 'none'

version: 1.0.0+1

environment:
  sdk: '>=3.1.2 <4.0.0'

dependencies:
  flutter:
    sdk: flutter

  firebase_auth: ^4.5.0
  firebase_core: ^3.8.1
  firebase_messaging: ^14.0.0
  firebase_storage: ^11.1.0
  firebase_core_platform_interface: ^5.3.1

  get: ^4.6.6
  shared_preferences: ^2.2.2
  email_validator: '^2.1.16'
  cloud_firestore: ^5.5.1
  geocoding: ^2.1.1
  geocode: ^1.0.3
  google_sign_in: ^6.1.5
  flutter_svg: ^0.22.0
  url_launcher: ^6.2.1
  flutter_map: ^5.0.0
  geoflutterfire2: ^2.0.0
  geolocator: ^10.0.1
  latlong2: ^0.9.0
  path: ^1.9.0
  image_picker: ^1.0.6
  timeago: ^3.2.2
  http: ^1.1.0

dev_dependencies:
  flutter_test:
    sdk: flutter
  flutter_launcher_icons: ^0.11.0
  flutter_lints: ^2.0.0

flutter_icons:
  android: "launcher_icon"
  ios: true
  image_path: "assets/icon/icon.png"
  remove_alpha_ios: true

flutter:
  uses-material-design: true

  assets:
    # Add assets from the images directory to the application.
    - assets/images/

**Comment by IshaqFarid280** [comment]
did you find the answer?
